R/bs-theme.R

```diff
@@ -397,40 +387,38 @@ bootswatch_bundle <- function(bootswatch, version) {
     )
   }
 
-  bootswatch_core <- sass_layer(
-    file_attachments = attachments,
-    defaults = list(
-      # Use local fonts (this path is relative to the bootstrap HTML dependency dir)
-      '$web-font-path: "font.css" !default;',
-      bootswatch_sass_file(bootswatch, "variables", version)
-    ),
-    rules = list(
-      bootswatch_sass_file(bootswatch, "bootswatch", version),
-      # For some reason sketchy sets .dropdown-menu{overflow: hidden}
-      # but this prevents .dropdown-submenu from working properly
-      # https://github.com/rstudio/bootscss/blob/023d455/inst/node_modules/bootswatch/dist/sketchy/_bootswatch.scss#L204
-      if (identical(bootswatch, "sketchy")) as_sass(".dropdown-menu{ overflow: inherit; }") else ""
+  sass_bundle(
+    bootswatch := sass_layer(
```

Comment:
Does the left side need a `!!`?

```suggestion
    !!bootswatch := sass_layer(
```

Reply:
No, this is intentional so that you can do `bs_remove(bs_theme(), "bootswatch")` and have it work regardless of the theme